examples/unixd

```diff
@@ -7,4 +7,5 @@
 # home_alias = "spn"
 # use_etc_skel = false
 # uid_attr_map = "spn"
-# gid_attr_map = "spn"
\ No newline at end of file
+# gid_attr_map = "spn"
+# selinux = true
```

Comment:
Could be worth a comment here for admins who will read this and assume "oh selinux is always enabled when I'm not using it". when the comment should be "If selinux is detected, relabel created home directories. If you are not using selinux then no action is taken".
Alternately, just remove it from this example since the default is sane.

Reply:
I think it's best to remove it so there's no confusion.